database_normalization/database_normalization.md

```diff
@@ -0,0 +1,370 @@
+<!--
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+module_template_version: 3.0.0
+language: en
+narrator: US English Female
+title: Database Normalization
+comment: Learn about the concept of normalization and why it's important for organizing complicated data in relational databases.
+long_description: Usually, data in a relational database like SQL is organized into multiple interrelated tables with as little data repetition as possible. This concept can be useful to apply in other areas as well, such as organizing data in .csvs or in data frames in R or Python.  This module teaches underlying data considerations and explains how data can be efficiently organized by introducing the concepts of one-to-many data relationships and normalization.
+estimated_time: 40 minutes
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Explain the significance of "one to many" data relationships and how these relationships affect data organization
+- Describe how a normalized database is typically organized
+- Explain how data can be linked between tables and define "primary keys" and "foreign keys"
```

Comment:
Question: are the terms "primary key" and "foreign key" specific to SQL or are they general information theory terms?

Reply:
I'd say they are "relational database" terms, e.g. Microsoft Access uses them, SQL uses them, and people use them "off label" even when they don't apply (e.g. Google BigQuery doesn't have foreign keys but we talk about "using xyz as a foreign key."

Reply:
Great! Good to know. Any chance this could fit in as a "behind the scenes" or "cool fact" or "history" box in the Key Vocabulary section?

Reply:
ooh I like it!  Will do!